Linking problem on Windows:

Looks like CMake can't find the Rizin library
```
    [4/4] Linking target _rizin.cp39-win_amd64.pyd
    FAILED: _rizin.cp39-win_amd64.pyd
    "c++"  -o _rizin.cp39-win_amd64.pyd _rizin.cp39-win_amd64.pyd.p/meson-generated_.._rizin_wrap.cxx.obj "-Wl,--allow-shlib-undefined" "-Wl,-O1" "-shared" "-Wl,--start-group" "-Wl,--out-implib=_rizin.cp39-win_amd64.dll.a" "C:\Users\runneradmin\AppData\Local\pypa\cibuildwheel\Cache\nuget-cpython\python.3.9.13\tools\python39.dll" "-lkernel32" "-luser32" "-lgdi32" "-lwinspool" "-lshell32" "-lole32" "-loleaut32" "-luuid" "-lcomdlg32" "-ladvapi32" "-Wl,--end-group"
    _rizin.cp39-win_amd64.pyd.p/meson-generated_.._rizin_wrap.cxx.obj:rizin_wrap.cxx:(.text+0x16cc): undefined reference to `rz_cons_set_raw'
    _rizin.cp39-win_amd64.pyd.p/meson-generated_.._rizin_wrap.cxx.obj:rizin_wrap.cxx:(.text+0x172c): undefined reference to `rz_cons_enable_highlight'
    _rizin.cp39-win_amd64.pyd.p/meson-generated_.._rizin_wrap.cxx.obj:rizin_wrap.cxx:(.text+0x17dc): undefined reference to `rz_cons_switchbuf'
    _rizin.cp39-win_amd64.pyd.p/meson-generated_.._rizin_wrap.cxx.obj:rizin_wrap.cxx:(.text+0x183c): undefined reference to `rz_cons_set_flush'
   ....
```
This looks quite similar to https://github.com/pypa/cibuildwheel/issues/1890 but the workaround didn't help.

_Originally posted by @XVilka in https://github.com/rizinorg/rz-bindgen/issues/68#issuecomment-2555510832_

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -19,7 +19,7 @@ environment = { PATH="/opt/rh/llvm-toolset-7.0/root/usr/bin:/opt/rh/llvm-toolset
 archs = ["x86_64"]
 
 [tool.cibuildwheel.windows]
-environment = { CMAKE_PREFIX_PATH="C:\\rizin" }
+environment = { CMAKE_PREFIX_PATH="C:/rizin" }
 archs = ["AMD64"]
 
 [tool.cibuildwheel.macos]

